backend/src/attraction/models.py

```diff
@@ -27,3 +38,24 @@ class Attraction(Base):
         nullable=False,
     )
     user = relationship("User", back_populates="attractions")
+
+    average_rating: Mapped[float] = column_property(
+        select(func.coalesce(func.avg(Review.rating), 0))
+        .where(Review.attraction_id == id)
+        .correlate_except(Review)
+        .scalar_subquery()
+    )
+
+    average_time_spent: Mapped[float] = column_property(
+        select(func.coalesce(func.avg(Review.time_spent), 0))
+        .where(Review.attraction_id == id)
+        .correlate_except(Review)
+        .scalar_subquery()
+    )
+
+    average_price: Mapped[float] = column_property(
+        select(func.coalesce(func.avg(Review.price), 0))
+        .where(Review.attraction_id == id)
+        .correlate_except(Review)
+        .scalar_subquery()
+    )
```

Comment:
average price ?

Reply:
Yes, because we want use a 5-point price scale such as $, $-$$, $$, $$-$$$, $$$

```
1 - $
2 - $-$$
3 - $$
4 - $$-$$$
5 - $$$
```